web/themes/joinup/prototype/html-prototype-sandbox/partials/listing-item--event-col-12.mustache

```diff
@@ -1,4 +1,4 @@
-<div class="listing__item listing__item--tile mdl-cell mdl-cell--12-col">
+<div class="listing__item listing__item--tile mdl-cell mdl-cell--12-col mdl-cell--9-col-wide">
```

Comment:
Partials naming has been changed due to changed related with grid:

- **listing-item--event-col-12.mustache** became **listing-item--event-col-9.mustache**

- **listing-item--event-col-4.mustache** became **listing-item--event-col-3.mustache**

I hope I understood your point well.